test_store.py

```diff
@@ -42,6 +47,15 @@ def test_remove_todo(db):
     assert task.done is False
 
 
+def test_remove_commits_to_db(db):
```

Comment:
being able to leave off the commit and the test still working because we're using an in memory database bugged me, so I added this test that made the commit() line required